pyHalo/preset_models.py

```diff
@@ -373,5 +383,63 @@ def ULDM(z_lens, z_source, log_mlow=6., log_mhigh=10., b_uldm=1.1, c_uldm=-2.2,
     realization_line_of_sight = pyhalo.render(['LINE_OF_SIGHT', 'TWO_HALO'], kwargs_model_field, nrealizations=1)[0]
     uldm_realization = realization_line_of_sight.join(realization_subs, join_rendering_classes=True)
 
+    if flucs: # add fluctuations to realization
+        ext = RealizationExtensions(uldm_realization)
+        lambda_dB = _de_broglie_wavelength(log10_m_uldm) # de Broglie wavelength in kpc
+        delta_kappa = _delta_sigma(z_lens,z_source,10**log_m_host,einstein_radius,lambda_dB) #amplitude of fluctuations
+
+        if flucs_args=={}:
+            raise Exception('Must specify fluctuation arguments, see realization_extensions.add_ULDM_fluctuations')
+
+        uldm_realization = ext.add_ULDM_fluctuations(de_Broglie_wavelength=lambda_dB,
+                                fluctuation_amplitude_variance=delta_kappa,
+                                fluctuation_size_variance=lambda_dB,
+                                shape=flucs_shape,
+                                args=flucs_args)
+
     return uldm_realization
 
+def _de_broglie_wavelength(log10_m_uldm,v=200):
+    '''
+    Returns de Broglie wavelength of the ultra-light axion in kpc.
+
+    :param log10_m_uldm: log(axion mass) in eV
+    :param v: velocity in km/s
+    '''
+    m_axion=10**log10_m_uldm
+    return 1.2*(1e-22/m_axion)*(100/v)
+
+def _delta_sigma(z_lens,z_source,m,rein,de_Broglie_wavelength):
```

Comment:
please put this in pyHalo.utilities (also doesn't need to be a private function)